doge-webapp/src/main/java/main/doge/service/TodoService.java

```diff
@@ -11,11 +12,17 @@
 @Service
 public class TodoService {
 
+    private final RestTemplate restTemplate;
+    private final String uri;
+
     @Autowired
-    private RestTemplate restTemplate;
+    public TodoService(RestTemplate restTemplate, @Value("${doge.service.uri}") String uri) {
```

Comment:
afaik you can inject this String as a `URL` and get startup time validation of the URL string